DuggaSys/microservices/duggaedService/deleteDuggaVariant_ms.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+date_default_timezone_set("Europe/Stockholm");
+
+// Include basic application services!
+include_once "../../../Shared/sessions.php";
+include_once "../../../Shared/basic.php";
+
+pdoConnect();
+session_start();
+
+$opt=getOP('opt');
+$vid=getOP('vid');
+
+if(strcmp($opt,"DELVARI")===0){
```

Comment:
I think we might want to check if the user has the right privileges to make the change.